library/argument/src/ap_option_with_parameter.e

```diff
@@ -93,9 +95,13 @@ feature -- Access
 	parameter: detachable G
 			-- Last value give to the option
 		require
-			was_found: was_found
+			was_found: was_found or has_default_value
 		do
-			Result := parameters.last
+			if parameters.is_empty then
```

Comment:
I think that what Berend had in mind (at least this is how his code works) is that if the option is not specified at all on the command-line (i.e. was_found is False), then we still have a default value. For example if --size is not specified, then it's as if you specified --size=20.